docs/Examples.md

```diff
@@ -190,16 +197,19 @@ metadata:
 spec:
   clusterMasterRef:
     name: cm
+  monitoringConsoleRef:
+    name: example_mc
 EOF
 ```
 
-The important parameter to note here is the `clusterMasterRef` field which points to the cluster master of the indexer cluster.
-Having a separate CR for cluster master gives us the control to define a size or StorageClass for the PersistentVolumes of the cluster master
-different from the indexers:
+Note that the `clusterMasterRef` field points to the cluster master for the indexer cluster. This example includes the `monitoringConsoleRef` parameter used to define a monitoring console pod. The monitoring console pod does not need to be running; the name can be predefined and the pod started later.
```

Comment:
@smohan-splunk I placed the `clusterMasterRef` into the CM, Index Cluster, and SHC examples per the comment you made in the prior PR. Starts at "This example includes..." I think that meets your goals.